src/core/frame.h

```diff
@@ -846,6 +846,7 @@ typedef struct QUIC_ACK_FREQUENCY_EX {
     QUIC_VAR_INT SequenceNumber;
     QUIC_VAR_INT PacketTolerance;
     QUIC_VAR_INT UpdateMaxAckDelay; // In microseconds (us)
+    QUIC_VAR_INT ReorderingThreshold;
     BOOLEAN IgnoreOrder;
     BOOLEAN IgnoreCE;
```

Comment:
These fields are no longer in the spec AFAICT:
```
ACK_FREQUENCY Frame {
  Type (i) = 0xaf,
  Sequence Number (i),
  Ack-Eliciting Threshold (i),
  Requested Max Ack Delay (i),
  Reordering Threshold (i),
}
```
Please remove them and update.

Reply:
Yes was just doing that, QUIC_ACK_FREQUENCY_EXTRAS also goes right cause we dont use it anyways